test/e2e/suite/plugin/install.go

```diff
@@ -60,24 +60,38 @@ var _ = Describe("notation plugin install", func() {
 		})
 	})
 
+	It("with content inside zip archive exceeds 256 MiB size limit", func() {
```

Comment:
nit: Please add test for zipbomb and zip slip. The zips
1. Zipbomb sample: https://www.bamsoftware.com/hacks/zipbomb/
2. ZipSlip sample: `zip zipslip.zip ../../../../../../../../tmp/evil.sh`

Reply:
Tests added.